srt-protocol/src/connection/mod.rs

```diff
@@ -371,35 +370,35 @@ impl DuplexConnection {
         ReceiverContext::new(
             &mut self.timers,
             &mut self.output,
-            &mut self.statistics,
+            &mut self.stats,
             &mut self.receiver,
         )
     }
 
-    fn debug(&self, tag: &str, now: Instant, debug: &impl Debug) {
+    fn debug(&self, tag: &str, debug: &impl Debug) {
         log::debug!(
             "{:?}|{:?}|{} - {:?}",
-            TimeSpan::from_interval(self.settings.socket_start_time, now),
+            TimeSpan::from_interval(self.settings.socket_start_time, Instant::now()),
             self.settings.local_sockid,
             tag,
             debug
         );
     }
 
-    fn info(&self, tag: &str, now: Instant, debug: &impl Debug) {
+    fn info(&self, tag: &str, debug: &impl Debug) {
         log::info!(
             "{:?}|{:?}|{} - {:?}",
-            TimeSpan::from_interval(self.settings.socket_start_time, now),
+            TimeSpan::from_interval(self.settings.socket_start_time, Instant::now()),
```

Comment:
I don't have a huge stake in this, but I would probably prefer this keeps with passing `now` for when we run the connection at non-realtime

Reply:
ah, good point